cpg_utils/hail.py

```diff
@@ -187,3 +187,36 @@ def image_path(suffix: str) -> str:
     prefix = os.getenv('CPG_IMAGE_REGISTRY_PREFIX')
     assert prefix
     return f'{prefix}/{suffix}'
+
+
+def authenticate_job(
+    job,
+    print_all_statements: bool = True,
+):
+    """
+    Takes a hail batch job, activates the appropriate service account
+
+    Once multiple environments are supported this method will decide
+    on which authentication method is appropriate
+
+    Parameters
+    ----------
+    job
+        * A hail BashJob
+    print_all_statements
+        * logging toggle
+
+    Returns
+    -------
+    None
+    """
+
+    # set the key path variable inside the job environment
+    job.env('AUTH_KEY_PATH', '/gsa-key/key.json')
```

Comment:
I don't have a strong opinion here, but could also just inline that in the `command`.